benchmarks/test_get_qsp_program.py

```diff
@@ -0,0 +1,94 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from benchq.algorithms.time_evolution import _n_block_encodings_for_time_evolution
+from benchq.problem_embeddings import get_qsp_program
+from benchq.problem_ingestion import (
+    generate_jw_qubit_hamiltonian_from_mol_data,
+    get_vlasov_hamiltonian,
+)
+from benchq.problem_ingestion.hamiltonian_generation import fast_load_qubit_op
+from benchq.problem_ingestion.molecule_instance_generation import (
+    generate_hydrogen_chain_instance,
+)
+
+SKIP_SLOW = pytest.mark.skipif(
+    os.getenv("SLOW_BENCHMARKS") is None,
+    reason="Slow benchmarks can only run if SLOW_BENCHMARKS env variable is defined",
+)
+
+
+def vlasov_test_case():
+    k = 2.0
+    alpha = 0.6
+    nu = 0.0
+    N = 2
+
+    evolution_time = 5
+    failure_tolerance = 1e-3
+
+    operator = get_vlasov_hamiltonian(k, alpha, nu, N)
+
+    n_block_encodings = _n_block_encodings_for_time_evolution(
+        operator, evolution_time, failure_tolerance
+    )
+
+    return pytest.param(operator, n_block_encodings, id="vlasov")
+
+
+def jw_test_case():
+    evolution_time = 5
+    failure_tolerance = 1e-3
+    n_hydrogens = 2
+
+    operator = generate_jw_qubit_hamiltonian_from_mol_data(
+        generate_hydrogen_chain_instance(n_hydrogens)
+    )
+
+    n_block_encodings = _n_block_encodings_for_time_evolution(
+        operator, evolution_time, failure_tolerance
+    )
+
+    return pytest.param(
+        operator,
+        n_block_encodings,
+        id=f"jw-{n_hydrogens}",
+    )
+
+
+def fast_load_test_cases():
+    def _load_hamiltonian(name):
+        return fast_load_qubit_op(
+            str(Path(__file__).parent / f"../examples/small_molecules/{name}.json")
```

Comment:
In a recent PR I have changed this path and deleted the `small_molecules.zip`, so perhaps it might be worth double checking if this line still works ;)

Reply:
🤦 
Can you please move this zip file to `data` ?
I thought I did that...